src/pvrclient-nextpvr.cpp

```diff
@@ -285,17 +285,31 @@ bool cPVRClientNextPVR::Connect()
                   {
                     if ( g_livestreamingmethod != RealTime)
                     {
-                      if (version >= 50001)
+                      if (g_livestreamingmethod == Transcoded)
+                      {
+                        delete m_timeshiftBuffer;
+                        m_timeshiftBuffer = new timeshift::TranscodedBuffer();
+                      }
+                      else if (version >= 50001)
                       {
                         g_livestreamingmethod = ClientTimeshift;
                       }
                       else
                       {
                         g_livestreamingmethod = RealTime;
-                        XBMC->QueueNotification(QUEUE_ERROR,"Upgrade to NextPVR version 5.0.1 or higher");
+                        XBMC->QueueNotification(QUEUE_ERROR,"NextPVR v5.0.1+ required for timeshift");
                       }
                     }
                   }
+                  else
+                  {
+                    if (g_livestreamingmethod == Transcoded)
+                    {
+                        g_livestreamingmethod = RealTime;
+                        XBMC->QueueNotification(QUEUE_ERROR,"NextPVR v5 required for transcoding");
```

Comment:
localization?

Reply:
The NextPVR backend setup is currently English only and only English forum support is provided.  I will create an issue to localize these  QueueNotifications along with the existing ones.

Reply:
Don't get how localization of the add-on  should be related to the backend add you are creating a UI string here. You do not pass something you get from the Backend to Kodi. So, add a new string to en_gb strings.po and that's it.

Reply:
In the amended update I was able to map all notifications to existing localization